### PR TITLE
deps: rm remark-gfm as uneeded, comments_async/comment: add <mark> ta…

### PR DESCRIPTION
**Pull request by phillimorland**
…g to moderator marked comments

### CODE REVIEW

File: adhocracy4/comments_async/static/comments_async/comment.jsx [hunk 1]
```diff
@@ -201,13 +200,21 @@ export default class Comment extends React.Component {
       }
       comment = (
         <div className={'a4-comments__text' + (this.state.anchored ? ' a4-comments__text--highlighted' : '')}>
```
Comment (phillimorland) [inline adhocracy4/comments_async/static/comments_async/comment.jsx:202]: we should change this to a4-comments__text--linked as that what the styling actually shows but we would need to change it everywhere so not sure about doing now?!

File: adhocracy4/comments_async/static/comments_async/comment.jsx [hunk 2]
```diff
@@ -201,13 +200,21 @@ export default class Comment extends React.Component {
...
-          <ReactMarkdown
-            disallowedElements={['h1', 'h2', 'h3', 'h4', 'input', 'table', 'thead', 'tr', 'th']}
-            unwrapDisallowed
-            plugins={[[gfm, { singleTilde: false }]]}
-          >
-            {content}
-          </ReactMarkdown>
+          {this.props.is_moderator_marked
+            ? <mark>
```
Comment (khamui) [inline adhocracy4/comments_async/static/comments_async/comment.jsx:204]: this seems to highlight/mark the entire line in cases where the comment is shorter. Is this what we want?

Comment (phillimorland) [reply adhocracy4/comments_async/static/comments_async/comment.jsx:204]: nope but really not sure how else to do it with the markdown, did have a play yesterday but it's not a big story, also I imagine a moderator marked comment that is amazing probable will be longer then 2-3 words hopefully

Comment (phillimorland) [reply adhocracy4/comments_async/static/comments_async/comment.jsx:204]: it's not ideal I know!

**Pull request merged**